.bazelversion

```diff
@@ -1,4 +1,4 @@
-7.1.1
+7.4.1
```

Comment:
this is problematic, will users be forced to upgrade Bazel as well? In that case it's a breaking change and hard to rollout. We should look at alternatives that work with older Bazel, and/or whether we can have some conditional logic using bazel_features so the new way is only enabled when the Bazel version supports it